pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -134,7 +133,7 @@ func (c *crtbLifecycle) reconcileSubject(binding *v3.ClusterRoleTemplateBinding)
 		return binding, nil
 	}
 
-	return nil, errors.Errorf("Binding %v has no subject", binding.Name)
+	return nil, fmt.Errorf("Binding %v has no subject", binding.Name)
```

Comment:
I appreciate that this is a search/replace, but...maybe we could fix this to indicate which kind of binding this is :-)

Reply:
Done!